packages/server/graphql/mutations/helpers/removeSlackAuths.ts

```diff
@@ -0,0 +1,46 @@
+import fromTeamMemberId from 'parabol-client/utils/relay/fromTeamMemberId'
+import getRethink from '../../../database/rethinkDriver'
+
+const removeSlackAuths = async (teamMemberIds: string | string[], removeToken: boolean) => {
+  const r = await getRethink()
+  const now = new Date()
+  const teamMemberIdsArr = Array.isArray(teamMemberIds) ? teamMemberIds : [teamMemberIds]
+  const {userId} = fromTeamMemberId(teamMemberIdsArr[0])
```

Comment:
this looks a little hairy because i'd assume I could call this function with a list of any teamMemberIds, but in fact I have to call the function with an array of teamMemberIds that all have the same userId.

to clean up, let's make the function signature `(userId: string, teamIds: string[], removeToken?: boolean)`